server/sonar-server/src/main/java/org/sonar/server/rule/RubyRuleService.java

```diff
@@ -49,7 +49,8 @@
  * @deprecated in 4.4 because Ruby on Rails is deprecated too !
  */
 @Deprecated
-public class RubyRuleService implements ServerComponent, Startable {
+@ServerSide
+public class RubyRuleService implements Startable {
```

Comment:
![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)